hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMCertStore.java

```diff
@@ -93,6 +114,8 @@ public void revokeCertificate(BigInteger serialID) throws IOException {
                scmMetadataStore.getStore().initBatchOperation();) {
         scmMetadataStore.getRevokedCertsTable()
             .putWithBatch(batch, serialID, cert);
+        scmMetadataStore.getValidSCMCertsTable().deleteWithBatch(batch,
```

Comment:
we only need this if the revoked cert is a scm cert.

Reply:
Done